tlspuffin/src/tls/seeds.rs

```diff
@@ -397,6 +397,51 @@ pub fn seed_successful12(client: AgentName, server: AgentName) -> Trace<TLSProto
     }
 }
 
+pub fn seed_successful12_forward(client: AgentName, server: AgentName) -> Trace<TLSProtocolTypes> {
```

Comment:
Add a comment explaining what this should do

Reply:
Done